pkg/plugin/cmd.go

```diff
@@ -442,11 +465,19 @@ func (o *DebugOptions) Run() error {
 		params := url.Values{}
 		params.Add("image", o.Image)
 		params.Add("container", containerID)
-		bytes, err := json.Marshal(o.Command)
+		var authStr string
+		registrySecret, err := o.CoreClient.Secrets(o.RegistrySecretNamespace).Get(o.RegistrySecretName, v1.GetOptions{})
+		if err != nil {
+			authStr = ""
```

Comment:
We can safely skip auth if the secret is not found, but in other conditions, this error should be properly handled.